zerver/openapi/openapi.py

```diff
@@ -256,9 +256,9 @@ def get_parameters_description(endpoint: str, method: str) -> str:
 
 def generate_openapi_fixture(endpoint: str, method: str, status_code: str = "200") -> List[str]:
     """Generate fixture to be rendered"""
-    fixture = []
+    fixture = [f"#### {status_code} Response(s)"]
```

Comment:
Response [should not](https://zulip.readthedocs.io/en/latest/translating/translating.html#capitalization) be capitalized here.